docs/ledger.md

```diff
@@ -1,5 +1,10 @@
+---
+id: ledger
+title: Ledger
+---
 # Ledger vault
-
+Connect the ledger device to the system in which signatory is running.
+Install tezos-wallet and tezos-baker apps from ledger live. 
```

Comment:
You might want to link to ledger live for downloads since it will make peoples lives easier